config/source/multi/600-config-scheduler.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2021 The Knative Authors
```

Comment:
seems like it should be 400-config-scheduler.yaml to be consistent

Reply:
Done